### DISCUSSION TITLE
Could Mark down support be added to the Notes function?

### DISCUSSION THREAD

**Discussion by ei8fdb** [magicbug/Cloudlog#3230] [Ideas / Feature Requests]
I use the notes function a lot, especially when I'm operating, or contesting. I find it mich easier to have useful linkes (ITU/CQ maps/links to maidenhead/log-lat converter tools) and IARU bandplans etc.

Right now Notes works well, it supports basic formatting - header/body text/bold, italic, underline, links. I dunno if this is markdown or HTML. 

Is there a possibltility it could support markdown elements like table? PDF URL embedding? checkboxes/task list? 

**How markdown looks in the Notes editor**
<img width="1069" alt="image" src="https://github.com/user-attachments/assets/173113bc-a745-46e6-9588-c03b91d7ec6a">

**How markdown looks in the Notes viewer**
<img width="868" alt="image" src="https://github.com/user-attachments/assets/b2dc98db-96df-4985-98e2-c3c5f4734140">

**Comment by magicbug** [comment]
I'm not against having markdown support, at the moment I don't think the current js UI supports it but will look into it!
